lib/internal/util.js

```diff
@@ -257,6 +257,18 @@ function promisify(original) {
 
 promisify.custom = kCustomPromisifiedSymbol;
 
+// The build in join is relatively slow (currently v8 6.0)
```

Comment:
I would replace this with something like: 'array.join in v8 6.0 is slower'

Also, adding a link to the issue in the comment would be beneficial.

Reply:
Also, typo? [built-in](https://en.wiktionary.org/wiki/built-in)